test/e2e/tests/tokens/nft/erc721-interaction.spec.js

```diff
@@ -51,19 +51,17 @@ describe('ERC721 NFTs testdapp interaction', function () {
         await driver.clickElement(
           '[data-testid="account-overview__activity-tab"]',
         );
-        const transactionItem = await driver.waitForSelector({
+        await driver.waitForSelector({
           css: '[data-testid="activity-list-item-action"]',
           text: 'Deposit',
         });
-        assert.equal(await transactionItem.isDisplayed(), true);
```

Comment:
anti-pattern: this causes race conditions when the transaction element is found and updated before we do the assertion, making the element become stale. See example error of this:

![Screenshot from 2024-10-17 19-57-08](https://github.com/user-attachments/assets/79cc04ee-60d9-45b9-9bbb-a984da0a1bf1)